DeepCat.js

```diff
@@ -149,6 +155,13 @@
 		$( '#searchform' ).submit();
 	}
 
+	/**
+	 * Replace "deepcat:" search terms with "incategory:" terms from DeepCat response
+	 *
+	 * @param {array} responses Category search results
+	 * @param {array} newSearchTerms Original search terms provided by the user
+	 * @returns {array} Modified newSearchTerms
```

Comment:
Although PhpStorm inserts `@returns` tag, the tag we are using is actually `@return`, see: https://www.mediawiki.org/wiki/Manual:Coding_conventions/JavaScript#Tags